ReactAndroid/build.gradle

```diff
@@ -293,30 +293,30 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'src/main/third-party/java/infer-annotations/', include: ['*.jar'])
-    compile 'javax.inject:javax.inject:1'
-    compile 'com.android.support:appcompat-v7:26.1.0'
-    compile 'com.facebook.fbui.textlayoutbuilder:textlayoutbuilder:1.0.0'
-    compile 'com.facebook.fresco:fresco:1.10.0'
-    compile 'com.facebook.fresco:imagepipeline-okhttp3:1.9.0'
-    compile 'com.facebook.soloader:soloader:0.5.1'
-    compile 'com.google.code.findbugs:jsr305:3.0.2'
-    compile 'com.squareup.okhttp3:okhttp:3.10.0'
-    compile 'com.squareup.okhttp3:okhttp-urlconnection:3.10.0'
-    compile 'com.squareup.okio:okio:1.14.0'
+    implementation fileTree(dir: 'src/main/third-party/java/infer-annotations/', include: ['*.jar'])
+    implementation 'javax.inject:javax.inject:1'
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.facebook.fbui.textlayoutbuilder:textlayoutbuilder:1.0.0'
+    implementation 'com.facebook.fresco:fresco:1.10.0'
+    implementation 'com.facebook.fresco:imagepipeline-okhttp3:1.10.0'
+    implementation 'com.facebook.soloader:soloader:0.5.1'
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
+    implementation 'com.squareup.okhttp3:okhttp:3.10.0'
+    implementation 'com.squareup.okhttp3:okhttp-urlconnection:3.10.0'
+    implementation 'com.squareup.okio:okio:1.14.0'
     compile 'org.webkit:android-jsc:r174650'
```

Comment:
Looks like forget to change here.

Reply:
I think maybe `api` is better.

Reply:
I mean most of them, like okhttp, fresco, appcompat. And also the compile is still exist.

Reply:
Changing compile to something else if breaking ReactAndroid build. So I think that it requires some refactoring to fix the issue.

Reply:
If you change it to api or implementation, what's the error ?

Reply:
Using `implementation` instead of `api` may break many things, since docs has show the usage too such as okhttp and fresco.

Reply:
@gengjiawen yes, you're right. I'll make changes to use api instead of compile